src/task_scl_eff_pot_hab.py

```diff
@@ -3,180 +3,200 @@
 from datetime import datetime, timezone
 from task_base import SCLTask
 
-import os
-import json
-service_account_key = os.environ.get("SERVICE_ACCOUNT_KEY")
-service_account_name = json.loads(service_account_key)["client_email"]
-credentials = ee.ServiceAccountCredentials(
-    service_account_name, key_data=service_account_key
-)
-ee.Initialize(credentials)
-
-# sumatra_aoi = ee.FeatureCollection("projects/SCL/v1/Panthera_tigris/sumatra_poc_aoi")
 
 class SCLPolygons(SCLTask):
     ee_rootdir = "projects/SCL/v1"
+    scale = 300
     inputs = {
         "structural_habitat": {
             "ee_type": SCLTask.IMAGECOLLECTION,
             "ee_path": "projects/SCL/v1/Panthera_tigris/geographies/Sumatra/hab/structural_habitat",
-            "maxage": 1,  # years
+            "maxage": 5,  # years
         },
         "hii": {
             "ee_type": SCLTask.IMAGECOLLECTION,
             "ee_path": "projects/HII/v1/sumatra_poc/hii",
-            "maxage": 1,
+            "maxage": 30,
+        },
+        "probability": {
+            "ee_type": SCLTask.IMAGECOLLECTION,
+            "ee_path": "projects/SCL/v1/Panthera_tigris/geographies/Sumatra/hab/probability",
+            "maxage": 30,
         },
         "historic_range": {
             "ee_type": SCLTask.IMAGE,
             "ee_path": "projects/SCL/v1/Panthera_tigris/source/Inputs_2006/hist",
-            "maxage": 1 / 365,
+            "maxage": 20,
```

Comment:
If you include `"static": 20` probably best to remove the maxage property -- see below inputs as well

Reply:
all static inputs have no maxage now